examples/GenerateMFEMMesh.cpp

```diff
@@ -0,0 +1,117 @@
+//                                PUMI Example
```

Comment:
Should this be a miniapp in `miniapps/meshing` or maybe part of `mesh-explorer` there?

Reply:
I took it out at the moment.